cm4-tool/tool.c

```diff
@@ -35,7 +35,6 @@ void rx_thread(void *arg)
 	while (!done) {
 		if (fread(&byte, 1, 1, f)) {
 			printf("%c", byte);
-			fflush(NULL);
```

Comment:
Why remove this `fflush`? It's needed, as there's no `\n`

Reply:
I don't remember why so I've restored it.